launcher/container_runner.go

```diff
@@ -79,6 +82,33 @@ const (
 	defaultRefreshJitter = 0.1
 )
 
+func fetchContainerImageSignatures(ctx context.Context, sdClient *signaturediscovery.Client, targetRepos []string, logger *log.Logger) []internal.Signature {
```

Comment:
not required, but it would be nice to be able to cache signatures so we don't have to fetch every time. This complicates things, so maybe add a TODO with an issue.